### ISSUE TITLE
[Feature Request] increase "screensaver" variety and options 

### ISSUE THREAD

**Issue by JayRugMan** [linuxmint/cinnamon-screensaver#306]
```
 * Cinnamon-Screensaver version 4.0.3
 * Distribution - Linux Mint 19.1
 * Graphics:  Device-1: Intel Xeon E3-1200 v2/3rd Gen Core processor Graphics driver: i915 v: kernel 
           Device-2: NVIDIA GF119 [NVS 310] driver: nvidia v: 390.77 
           Display: x11 server: X.Org 1.19.6 driver: modesetting,nvidia unloaded: fbdev,nouveau,vesa 
           resolution: 1920x1080~60Hz, 1920x1080~60Hz 
           OpenGL: renderer: NVS 310/PCIe/SSE2 v: 4.6.0 NVIDIA 390.77
 * 64 bit
```

**Issue**
I recently installed Mint 19.1 (have been on 19.0 for a while on another computer and loving it) and went to set some screensavers. Lo and behold, they are no more. I read the release notes and see that the xscreensaver hack was deprecated, but could be installed manually. I did that - and hate the login screen!! I would rather use the login screen native to Cinnamon on Mint 19.1, but with some decent, animated screensavers.

**How to rerpoduce**
1- Install Linux Mint 19.1 Cinnamon
2- go to System Settings > Preferences - Screensavers > 
3- There are only two tabs, none of which have options for the screensavers available in previous versions (including LM 19.0 )

**Expected behavior**
Are there plans to increase variety of "screensaver" options - an app that works better than a hack (like with xscreensaver), but has more than just the floating time? 

What are the best alternative options if not? 

Let me know if additional information is needed.

**Comment by pbs-dg** [comment]
I confirm this issue. 
It is such a basic feature I can't imagine how can it be broken.

**Comment by monoprix** [comment]
Hi, I won't lie, I miss xscreensaver. I understand that complications have arisen from the integration of xscreensaver with the lock screen, but I do hope you find a solution. As always, many, many thanks for all of your hard work, LM team.

**Comment by JayRugMan** [comment]
Does this mean Cinnamon isn't going to develop a screen saver that works with the lock screen? 

Mint 19.1 Mate has screen savers, but it's not cinnamon - and the lock screen is lack-luster at best. 

Any hope to be offered here, or are we just SOL on this?

**Comment by ctfolmar** [comment]
I keep a small desktop with my TV to use as a DVD and music player.  I like to play music on rythmbox while the screensaver shows pictures on the TV screen, particularly when we have guests. It keeps the TV from being this gaping black hole in the middle of the wall.  I currently have Mint 18.3 on that computer because it does just that perfectly. Would it be too much trouble to add a simple slideshow to the Cinnamon screensaver?  It doesn't have to be anything fancy, just cycle through pictures in a folder similar to a digital picture frame.  Thank you for your consideration.

**Comment by smurphos** [comment]
Pending this issue being addressed here is a shell script that facilitates a 
 configurable lock screen picture slideshow for Cinnamon - https://github.com/smurphos/nemo_actions_and_cinnamon_scripts#lock-screen-slideshow

**Comment by ctfolmar** [comment]
Thank you very much

Christopher Todd

**Comment by mtwebster** [comment]
What's wrong with using Cinnamon's background slideshow mechanism?

**Comment by smurphos** [comment]
> What's wrong with using Cinnamon's background slideshow mechanism?

If it offered a setting to kick in only when the screensaver was active it would be great.
And a setting to use an alternative backgrounds source directory when the screensaver was active for those users who want a slideshow when the screensaver isn't active, but a different set of images when the screen saver is active.

**Comment by JayRugMan** [comment]
Hi All - First of all, Mint is AWESOME and always getting better. I used it at work and at home. The screensavers are still lacking. Are there any good options for Cinnamon _yet_?

**Comment by mfreeman72** [comment]
If I'm remembering correctly, they removed the animated screensavers quite a while ago, largely because they felt they were unnecessary. I personally disagree. They may not be *functionally* necessary, but are necessary for artistic and personal expression purposes, and to give people something nice/artistic/entertaining to look at. I often used my computer as a form of artistic expression when not in use, and I really miss that functionality. I could, of course, install xscreensaver, but then the lock screen becomes hideous. It would be *so* nice to have those animated screensavers back. I'm not holding my breath, though. I have serious doubts that the devs share my opinions on aesthetics and the importance of personal artistic expression with your computer.

**Comment by JayRugMan** [comment]
I agree with @mfreeman72 100%. If there are no more screensavers because they are "no longer necessary" then maybe there shouldn't be transparent terminals or even desktop backgrounds and desktop icons. With Mint 21.1, they added some nuanced folder colors, which don't seem necessary either. So, what's the issue with having both awesome animated "screensavers" _and_ a pleasant-looking login screen? Why do we have to give one up for the other. It seems contemptuous of the developers to make user choose. Maybe they don't "save" screens any more, that could have been done with making the screen blank when idle. The animations added aesthetic enjoyment to the system when not in use. I'd love to get these back without having to sacrifice the aesthetics of the lock screen.

**Comment by JayRugMan** [comment]
## Update:
I found an answer here:
https://forums.linuxmint.com/viewtopic.php?p=1077179#p1077179

I'm currently on Linux Mint 21.1 x86_64, Dell G15 5520, 5.15.0-60-generic, Cinnamon 5.6.7 and this works fine while not messing up the login or lock screen. 

I'd say this issue is resolved.

**Comment by mfreeman72** [comment]
How can you say that doesn't mess up the login or lock screen?! It turns it completely off! That's not at all what I would call a solution to that problem.

**Comment by JayRugMan** [comment]
@mfreeman72 - Ugh... you're right. When I had everything installed and locked my screen manually, the login screen was normal... I thought that was enough... But alas, when I let the screensaver kick on, then lock my screen, it was as you say - back to the crappy login screen.. Why oh why. 

needless to say, almost there isn't good enough. I'm re-opening this issue.

**Comment by mfreeman72** [comment]
For the Github "stale-bot": This issue is still relevant to newer versions, and I don't see that changing in the near future, unfortunately.
